packages/core/strapi/lib/commands/opt-out-telemetry.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const { resolve } = require('path');
+const fse = require('fs-extra');
+const chalk = require('chalk');
+const fetch = require('node-fetch');
+const machineID = require('../utils/machine-id');
+
+const readPackageJSON = async path => {
+  try {
+    const packageObj = await fse.readJson(path);
+    const uuid = packageObj.strapi.uuid;
+
+    return [uuid, packageObj];
```

Comment:
since uuid is in the package why do you return two separate values? the function name is a bit confusing I think

Reply:
Habit of separate vars for everything 😅 No real reason, just nicer for my subjective eye

Reply:
we usually use object instead of arrays in the codebase for this then :)

Reply:
Object it is